sycl/doc/EnvironmentVariables.md

```diff
@@ -77,6 +56,52 @@ Assuming a filter has all three elements of the triple, it selects only those de
 
 Note that all device selectors will throw an exception if the filtered list of devices does not include a device that satisfies the selector. For instance, `SYCL_DEVICE_FILTER=cpu,level_zero` will cause `host_selector()` to throw an exception. `SYCL_DEVICE_FILTER` also limits loading only specified plugins into the SYCL RT. In particular, `SYCL_DEVICE_FILTER=level_zero` will cause the `cpu_selector` to throw an exception since SYCL RT will only load the `level_zero` backend which does not support any CPU devices at this time. When multiple devices satisfy the filter (e..g, `SYCL_DEVICE_FILTER=gpu`), only one of them will be selected.
 
+## Controlling DPC++ Level Zero Plugin
```

Comment:
If we really want to have Level Zero variables into its own category then that must be split to "production" and "debugging" as well. There are ones here that aren't intended to be used by the end user, e.g.

> | `SYCL_PI_LEVEL_ZERO_USE_COPY_ENGINE_FOR_D2D_COPY` (experimental) | Integer | Allows the use of copy engine, if available in the device, in Level Zero plugin for device to device copy operations. The default is 0. This option is experimental and will be removed once heuristics are added to make a decision about use of copy engine for device to device copy operations. |

Reply:
@smaslov-intel could you please somehow mark which variables go to which category, I'll make that change.

Reply:
I'd name all of these below as "debugging":

SYCL_PI_LEVEL_ZERO_MAX_COMMAND_LIST_CACHE
SYCL_PI_LEVEL_ZERO_USM_ALLOCATOR
SYCL_PI_LEVEL_ZERO_BATCH_SIZE
SYCL_PI_LEVEL_ZERO_FILTER_EVENT_WAIT_LIST
SYCL_PI_LEVEL_ZERO_USE_COPY_ENGINE
SYCL_PI_LEVEL_ZERO_USE_COPY_ENGINE_FOR_D2D_COPY
SYCL_PI_LEVEL_ZERO_DEVICE_SCOPE_EVENTS